Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -4075,14 +4075,8 @@ class LowerToAir {
         }
 
         case Const128: {
-            auto value = m_value->as<Const128Value>()->value();
-            auto a = tmpForType(Int64);
-            auto result = tmp(m_value);
-            append(MoveZeroToVector, result);
-            append(Move, Arg::bigImm(value.u64x2[0]), a);
-            append(Air::VectorReplaceLaneInt64, Arg::imm(0), a, result);
-            append(Move, Arg::bigImm(value.u64x2[1]), a);
-            append(Air::VectorReplaceLaneInt64, Arg::imm(1), a, result);
+            // We expect that the moveConstants() phase has run, and any constant vector referenced from stackmaps get fused.
```

Comment:
Should we assert not reached?

Reply:
We are not moving Const128 for zero case (as the same to ConstFloat / ConstDouble) since vxorps is faster and compact. So we need this `MoveZeroToVector`.